# phenotech

## About

Here you can find supplementary information for the ddRAD and hyRAD methods and raw data used in:

Lang, PLM <i>et al.</i> 2020, <b>Hybridization ddRAD-sequencing for population genomics of non-model plants using highly degraded historical specimen DNA</b>, Molecular Ecology Resources, [DOI:10.1111/1755-0998.13168](https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13168). 

## Protocols

All information is stored in writing-protected google sheets. The protocols contain macros for calculation of chemicals needed based on the number of samples. To be able to use these, click on the link to open the google sheets, and make your own copy of the document. 

<b>Oligo sequences</b> - [click](https://docs.google.com/spreadsheets/d/1PE-4ez0dlBuDm6VDlCPDjMJeLDjw5XQi0hW6Z0YECv0/edit#gid=706044524) to open the document in another browser window.

<b>Wetlab protocols</b> - [click](https://docs.google.com/spreadsheets/d/1kDV4ra6sYo5lCAtQFderUGwbjtkKvBc4xmXWhoXwS2Q/edit#gid=1446787502) to open the document in another browser window.
This includes the protocols for DNA extraction using CTAB, ddRAD library preparation, hyRAD library capture (all 96-well-plate optimized) and preparation of hyRAD baits. Recipes for buffers and magnetic beads are provided as well, as is information on reagent providers. 

## Data accessibility

All **sequencing reads** generated as part of this project are publicly available under project PRJEB36294 at ENA - [click](https://www.ebi.ac.uk/ena).


## Feedback
If you find errors in the resources provided here, or have other general feedback, please share â€“ we are happy to hear from you! 

Main contact: patricialmlang [at] gmail.com

Other contacts can be found in the paper.
